Sources/MulticastDelegate.swift

```diff
@@ -98,13 +98,10 @@ public class MulticastDelegate<T> {
  *
  *  - parameter left:   The multicast delegate
  *  - parameter right:  The delegate to be added
- *
- *  - returns: The `MulticastDelegate` instance with the `right` delegate parameter added
  */
-public func +=<T>(left: MulticastDelegate<T>, right: T) -> MulticastDelegate<T> {
+public func +=<T>(left: MulticastDelegate<T>, right: T) {
```

Comment:
This warning is erroneous and should be ignored... Swift requires that there _isn't_ any space  around this...